packages/expo/package.json

```
@@ -69,6 +71,7 @@
     "@expo/cli": "0.19.8",
     "@expo/config": "10.0.2",
     "@expo/config-plugins": "9.0.4",
+    "@expo/fingerprint": "0.11.0",
```

This version must match the version that is used in the monorepo. It should fix CI failures 😉 
```suggestion
    "@expo/fingerprint": "0.11.1",
```